README.md

```diff
@@ -70,7 +70,7 @@ Enter number:
   - When using Scala 2.11, you can use the Scala swing 2.0.x releases on JDK 6 or newer.
   - Scala 2.12 requires you to use JDK 8 (that has nothing to do with scala-swing).
 
-The reason to have two versions is to allow for binary incompatible changes. Also, some java-swing classes were generified in JDK 7 (see [SI-3634](https://issues.scala-lang.org/browse/SI-3634)) and require the scala-swing soruces to be adjusted.
+The reason to have two versions is to allow for binary incompatible changes. Also, some java-swing classes were generified in JDK 7 (see [SI-3634](https://issues.scala-lang.org/browse/SI-3634)) and require the scala-swing sources to be adjusted.
```

Comment:
mention a 2.1.x branch in the version paragraph above?

Reply:
There is no 2.1.x branch yet, though. Should I create one and rebalance the PR so we keep 2.0.x? Perhaps we can rename the branch to 2.x, although I don't know what consequencing a branch rename has w.r.t. to git?

Reply:
(Aaaaa - something is off in GitHub rendering of the conservation; the comments are correctly aligned only when using the 'Files changed' tab, and somehow my additional comments are not shown. So I put them here again:)

There is no 2.1.x branch yet, though. Should I create one and rebalance the PR so we keep 2.0.x? Perhaps we can rename the branch to 2.x, although I don't know what consequencing a branch rename has w.r.t. to git?

Reply:
My bad, I'd been thinking this new release was less compatible with existing than it is. No need for a branch, just mention that 2.0x supports 2.11, 2.12 and 2.13

Reply:
To clarify -  the version _will be_ 2.1.0 because it's not binary compatible with 2.0.x. I will draft an update for the README tomorrow and amend the PR.